src/Entity/AbstractTranslatableEntity.php

```diff
@@ -46,4 +44,13 @@ private function removeTranslationIfEmpty(string $locale): void
             $this->removeTranslation($translation);
         }
     }
+
+    /** @return Collection */
+    abstract public function getTranslations();
```

Comment:
can't we typehint the return with `Collection` ?

Reply:
@Remg, I can't do it, because they must respect the vendor Trait method signatures 
```
Compile Error: Declaration of A2lix\I18nDoctrineBundle\Doctrine\ORM\Util\Translatable::getTranslations() must be compatible with AppBundle\Entity
  \AbstractTranslatableEntity::getTranslations(): Doctrine\Common\Collections\Collection
```